cmake/kokkos_test_cxx_std.cmake

```diff
@@ -35,9 +35,18 @@ FUNCTION(kokkos_set_cxx_standard_feature standard)
     GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
   ENDIF()
 
-  IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
-    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
-  ENDIF()
+    IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+          IF (KOKKOS_CXX_COMPILER_ID STREQUAL "NVIDIA")
+                  MESSAGE(STATUS "nvcc_wrapper does not support TARGET_COMPILE_FEATURES")
+                        GLOBAL_SET(KOKKOS_CXX_STANDARD_FEATURE "")
+                            ELSE()
+                                    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
+                                        ENDIF()
+                                          ENDIF()
+
+  #  IF(NOT ${FEATURE_NAME} IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+  #    MESSAGE(FATAL_ERROR "Compiler ${KOKKOS_CXX_COMPILER_ID} should support ${FEATURE_NAME}, but CMake reports feature not supported")
+  #  ENDIF()
```

Comment:
This looks different from `develop`. Is this change intended?

Reply:
I think this was updated in develop after we created the incremental-testing branch.

Reply:
It seems that this change is also not present looking at all the commits in common, but it was in one of the recent comments. Again, cleaning up the history makes reviewing so much easier.

Reply:
where is this coming from? 
Why was this pushed here again.